odbc/src/handles.rs

```diff
@@ -1,12 +1,15 @@
+use crate::util::*;
 use std::{collections::HashSet, sync::RwLock};
 
-use odbc_sys::Integer;
+use odbc_sys::{Integer, SmallInt, SqlReturn, WChar};
 
 #[derive(Debug)]
 pub enum MongoHandle {
     Env(RwLock<Env>),
     Connection(RwLock<Connection>),
     Statement(RwLock<Statement>),
+    #[allow(dead_code)]
+    Descriptor(RwLock<Descriptor>),
```

Comment:
what's the reason for adding this if it's just dead code? I was under the impression that we were intentionally omitting the descriptor handle type until we had evidence that it was needed (though perhaps my understanding is stale)

Reply:
Some unimplemented functions, like `SQLSetDescField`,  specifically take descriptor handles as input, so we need a `Descriptor` variant in order to record errors for those functions